src/image-tags.js

```diff
@@ -54,6 +54,34 @@ function pickImage(pageType, data, url) {
   }
 }
 
+function pickAttributionFromCard (story, cardId) {
```

Comment:
we have already one function called `pickImageFromCard` . Can we return the Image attribution also from this function?